src/audio_core/adsp/apps/audio_renderer/audio_renderer.cpp

```diff
@@ -96,6 +97,15 @@ void AudioRenderer::SetCommandBuffer(s32 session_id, CpuAddr buffer, u64 size, u
     command_buffers[session_id].reset_buffer = reset;
 }
 
+void AudioRenderer::PostDSPClearCommandBuffer() noexcept {
+    command_buffers[0].buffer = 0;
+    command_buffers[0].size = 0;
+    command_buffers[0].reset_buffer = false;
+    command_buffers[1].buffer = 0;
+    command_buffers[1].size = 0;
+    command_buffers[1].reset_buffer = false;
```

Comment:
```suggestion
    for (u32 i = 0; i < MaxRendererSessions; i++) {
        command_buffers[i].buffer = 0;
        command_buffers[i].size = 0;
        command_buffers[i].reset_buffer = false;
    }
```
(or a ranged loop `auto& command_buffer : command_buffers`, whichever you prefer)